dask/compatibility.py

```diff
@@ -48,6 +52,43 @@ def _getargspec(func):
     def _getargspec(func):
         return inspect.getargspec(func)
 
+    if sys.version_info[1] <= 6:
+        class BZ2File(bz2.BZ2File, BufferedIOBase):
+            def __init__(self, *args, **kwargs):
+                super(BZ2File, self).__init__(*args, **kwargs)
+
+            def readable(self):
+                return 'r' in self.mode
+
+            def writable(self):
+                return 'w' in self.mode
+
+            def seekable(self):
+                return self.readable()
```

Comment:
Is this accurate?  Are bz2 files seekable?  Does this mean random-access seekable?